src/data/periods.ts

```diff
@@ -5,25 +5,29 @@ import { Duration } from '../util';
 export type Periods = {
 	diff: Duration | null,
 	label: string,
+	period_label: string,
 	value: Duration,
 	interval: string,
 }[];
 
 export const periods: Periods = [
 	{
 		label: __( '7 Days', 'altis' ),
+		period_label: __( '7-Day', 'altis' ),
 		value: 'P7D',
 		diff: 'P7D',
 		interval: '4h',
 	},
 	{
 		label: __( '30 Days', 'altis' ),
+		period_label: __( '30-Day', 'altis' ),
```

Comment:
Should this not be `30-Days`?
```suggestion
		period_label: __( '30-Days', 'altis' ),
```